<launch>

    <node name="controller_node" pkg="pp_pkg" type="pp_node" output="screen">
        <rosparam command="load" file="$(find pp_pkg)/config/default.yaml" />
    </node>

</launch>